front/reducers/itinerary-reducer.js

```diff
@@ -0,0 +1,20 @@
+const defaultState = {
+  wanderSpots: []
+};
+
+const itineraryReducer = (state=defaultState, action) => {
+  switch(action.type) {
+    case 'GET_WANDERSPOTS': {
+      return {...state}
+      break;
+    }
+    case 'ADD_MARKERS_FULFILLED': {
```

Comment:
Try explaining what you're trying to do with the 'ADD_MARKERS...' and I'll see if I can explain what's going on.